src/main/java/org/websoso/WSSServer/controller/FeedController.java

```diff
@@ -97,4 +100,16 @@ public ResponseEntity<FeedGetResponse> getFeed(Principal principal,
                 .body(feedService.getFeedById(user, feedId));
     }
 
+    @GetMapping("/popular")
+    public ResponseEntity<PopularFeedsGetResponse> getPopularFeeds(Principal principal) {
+        if (principal == null) {
+            return ResponseEntity
+                    .status(OK)
+                    .body(popularFeedService.getPopularFeeds(null));
+        }
+        User user = userService.getUserOrException(Long.valueOf(principal.getName()));
+        return ResponseEntity
+                .status(OK)
+                .body(popularFeedService.getPopularFeeds(user));
+    }
```

Comment:
p3; 혹시 이런식으로 바꾸는건 어떠실련지요? 분기처리를 해야 하는 다른 로직들(FeedController, UserNovelController 등)에서 위와 같이 쓰고 있어 제안해봅니다!
```suggestion
    @GetMapping("/popular")
    public ResponseEntity<PopularFeedsGetResponse> getPopularFeeds(Principal principal) {
        User user = principal == null ? null : userService.getUserOrException(Long.valueOf(principal.getName()));
        return ResponseEntity
                .status(OK)
                .body(popularFeedService.getPopularFeeds(user));
    }
```

Reply:
오호 삼항 연산자를 통해 간소화하는 코드네요
반영하겠습니다!